srcpkgs/php/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'php'
 pkgname=php
 version=7.4.14
-revision=5
+revision=6
```

Comment:
https://github.com/void-linux/void-packages/pull/32899 touches php as well, please bundle the changes into this PR.